### PR TITLE
updated graph

### LINKED ISSUE #196: Diagram out of date
The Diagram on the landing page is missing a few RFC pages and we can also link to the W3C DID and VC specs.

### PR DESCRIPTION
**Pull request by woutslakhorst**
closes #196

### CODE REVIEW

File: .gitbook/source/spec-relations.dot [hunk 1]
```diff
@@ -26,6 +29,8 @@ digraph D {
         RFC7662_EXT [label = "RFC7662\nOAuth 2.0 Token Introspection", href = "https://tools.ietf.org/html/rfc7662", fillcolor = azure2, style = filled]
         RFC7523_EXT [label = "RFC7523\nJSON Web Token (JWT) Profile\nfor OAuth 2.0 Client Authentication and Authorization Grants", href = "https://tools.ietf.org/html/rfc7523", fillcolor = azure2, style = filled,]
         RFC7515_EXT [label = "RFC7515\nJSON Web Signature (JWS)", href = "https://tools.ietf.org/html/rfc7515", fillcolor = azure2, style = filled]
+        DID_EXT [label = "W3C DID\nDecentralized Identifiers (DIDs) v1.0", href = "https://www.w3.org/TR/did-core/", fillcolor = azure2, style = filled]
+        VC_EXT  [label = "W3C VC\nVerifiable Credentials Data Model v1.1", href = "https://www.w3.org/TR/vc-data-model/", fillcolor = azure2, style = filled]
```
Comment (gerardsn) [inline .gitbook/source/spec-relations.dot:33]: labels mention a minor version while the links point to v1.x

File: .gitbook/source/spec-relations.dot [hunk 2]
```diff
@@ -18,6 +18,9 @@ digraph D {
         RFC012 [label = "RFC012\nNuts Organization Credential",href = "rfc012-nuts-organization-credential"];
         RFC013 [label = "RFC013\nVerifiable Credential IRMA Proof",href = "rfc013-vc-irma-proof"];
         RFC014 [label = "RFC014\nNuts Authorization Credential",href = "rfc014-authorization-credential"];
+        RFC015 [label = "RFC015\nNode Identity",href = "rfc015-node-identity"];
+        RFC016 [label = "RFC016\nLamport Clock",href = "rfc016-lamport-clock"];
```
Comment (gerardsn) [inline .gitbook/source/spec-relations.dot:22]: merged into RFC004

**Review by gerardsn — CHANGES_REQUESTED** [review]

File: .gitbook/source/spec-relations.dot (hunk 1 above)
Comment (woutslakhorst) [reply .gitbook/source/spec-relations.dot:33]: The site clearly states V1.1?
Screenshot 2022-09-13 at 11 57 28

File: .gitbook/source/spec-relations.dot (hunk 2 above)
Comment (woutslakhorst) [reply .gitbook/source/spec-relations.dot:22]: removed

File: .gitbook/source/spec-relations.dot (hunk 1 above)
Comment (gerardsn) [reply .gitbook/source/spec-relations.dot:32]: https://www.w3.org/TR/2022/REC-vc-data-model-20220303/ is the permalink of (the current version of) v1.1

We now link https://www.w3.org/TR/vc-data-model/ that currently points to v1.1, but used to point to v1.0 and maybe to v1.2 in the future?
There is also a v2 in the making on https://www.w3.org/TR/vc-data-model-2.0/

Comment (woutslakhorst) [reply .gitbook/source/spec-relations.dot:32]: Then we use the permalink

File: .gitbook/source/spec-relations.dot [hunk 3]
```diff
@@ -26,6 +28,8 @@ digraph D {
         RFC7662_EXT [label = "RFC7662\nOAuth 2.0 Token Introspection", href = "https://tools.ietf.org/html/rfc7662", fillcolor = azure2, style = filled]
         RFC7523_EXT [label = "RFC7523\nJSON Web Token (JWT) Profile\nfor OAuth 2.0 Client Authentication and Authorization Grants", href = "https://tools.ietf.org/html/rfc7523", fillcolor = azure2, style = filled,]
         RFC7515_EXT [label = "RFC7515\nJSON Web Signature (JWS)", href = "https://tools.ietf.org/html/rfc7515", fillcolor = azure2, style = filled]
+        DID_EXT [label = "W3C DID\nDecentralized Identifiers (DIDs) v1.0", href = "https://www.w3.org/TR/did-core/", fillcolor = azure2, style = filled]
```
Comment (gerardsn) [inline .gitbook/source/spec-relations.dot:31]:
```suggestion
        DID_EXT [label = "W3C DID\nDecentralized Identifiers (DIDs) v1.0", href = "https://www.w3.org/TR/2022/REC-did-core-20220719/", fillcolor = azure2, style = filled]
```

**Review by gerardsn — APPROVED** [review]

**Pull request merged**